src/host/ft_host/API_DimensionsTests.cpp

```diff
@@ -2,7 +2,9 @@
 // Licensed under the MIT license.
 
 #include "precomp.h"
-
+using namespace WEX::TestExecution;

```

Comment:
I'd prefer if there was a newline separating out the `#include`s from the `using`s.

Reply:
@adiviness I wanted to ask feedback on the changes made.